Gemfile

```diff
@@ -0,0 +1,31 @@
+source "https://rubygems.org"
+
+gem "bcrypt", "~> 3.1.7"
+gem "bootsnap", require: false
+gem "cancancan"
+gem "importmap-rails"
+gem "jbuilder"
+gem "pg", "~> 1.1"
+gem "puma", ">= 5.0"
+gem "rails", "~> 7.2.1", ">= 7.2.1.1"
+gem "sidekiq"
+gem "sprockets-rails"
+gem "stimulus-rails"
+gem "turbo-rails"
+gem "tzinfo-data", platforms: %i[ windows jruby ]
+
+group :development, :test do
+  gem "brakeman", require: false
+  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
+  gem "dotenv-rails"
+  gem "rubocop-rails-omakase", require: false
+end
+
+group :development do
+  gem "web-console"
+end
+
+group :test do
+  gem "capybara"
+  gem "selenium-webdriver"
```

Comment:
Do we need these gems for test purposes? If not, let's remove them

Reply:
Updated apu. Removed the gems not required for this PR. Thank you.